react-exercise/src/usersDBClient.ts

```diff
@@ -0,0 +1,52 @@
+import { User } from "./types/user";
+
+const API_URL = "http://localhost:4000/users";
+
+// Fetch all users
+export const fetchUsers = async (): Promise<User[]> => {
```

Comment:
We are usually using "Traditional Function Declaration" for functions that are not inside components - async function fetchUsers(): Promise<User[]> {}